pyfar/dsp/filter/gammatone.py

```diff
@@ -97,9 +97,9 @@ class GammatoneBands():
         >>>
         >>> y = GFB.reconstruct(real, imag)
         >>> plt.figure()
-        >>> ax = pf.plot.time_freq(
-        ...     y, label="manipulated and reconstructed and impulse")
-        >>> ax[0].set_xlim(0, .02)
+        >>> ax = pf.plot.time_freq(y, unit='ms',
+        ...     label="manipulated and reconstructed and impulse")
```

Comment:
indentation level, please check for flake 8 conformity

Reply:
When indenting this, flake8 fails as the line is too long. Should I indent as much as possible?

Reply:
In that case, you could use more lines, e.g., by by starting a new line right after `pf.plot.time_freq(`

Reply:
And then just indent 4 spaces?

Reply:
The function name and the label are too long otherwise

Reply:
yes, one tab (4 spaces) are the usual indent if splitting directly after `(`